base-notebook/start.sh

```diff
@@ -142,6 +143,6 @@ else
 
     # Execute the command
     run-hooks /usr/local/bin/before-notebook.d
-    echo "Executing the command: ${cmd[@]}"
+    echo "Executing the command:" "${cmd[@]}"
```

Comment:
This was a surprising change to me, do you understand what motivates it in shellcheck?

Reply:
This is the error with some explanation:
https://github.com/koalaman/shellcheck/wiki/SC2145